charts/datadog/README.md

```diff
@@ -733,7 +733,7 @@ helm install <RELEASE_NAME> \
 | datadog.kubeStateMetricsEnabled | bool | `false` | If true, deploys the kube-state-metrics deployment |
 | datadog.kubeStateMetricsNetworkPolicy.create | bool | `false` | If true, create a NetworkPolicy for kube state metrics |
 | datadog.kubelet.agentCAPath | string | /var/run/host-kubelet-ca.crt if hostCAPath else /var/run/secrets/kubernetes.io/serviceaccount/ca.crt | Path (inside Agent containers) where the Kubelet CA certificate is stored |
-| datadog.kubelet.coreCheckEnabled | bool | `false` |  |
+| datadog.kubelet.coreCheckEnabled | bool | `true` |  |
```

Comment:
The option description doesn't seems to be extracted properly from the `values.yaml` file.
could you investigate why and fix it? it might be an indentation issue.

Reply:
Thanks for catching this, the name of the variable did not match in `values.yaml` so the description didn't get populated 🙇